src/useLocalStore.ts

```diff
@@ -0,0 +1,22 @@
+import { observable } from "mobx"
+import { useState } from "react"
+
+import { useDeprecated } from "./utils"
+import { useAsObservableSource } from "./useAsObservableSource"
+
+export function useLocalStore<TStore extends Record<string, any>>(initializer: () => TStore): TStore
+export function useLocalStore<TStore extends Record<string, any>, TSource extends object>(
+    initializer: (source: TSource) => TStore,
+    current: TSource
+): TStore
+export function useLocalStore<TStore extends Record<string, any>, TSource extends object>(
+    initializer: (source?: TSource) => TStore,
+    current?: TSource
+): TStore {
+    if ("production" !== process.env.NODE_ENV)
```

Comment:
Unnecessary double-check I suppose? It's done in `useDeprecated` already.

Reply:
Will fix

Reply:
Updated, left it here but removed the other one, so that the messages themselves don't end up in the prod builds.